src/applications/gi-sandbox/reducers/autocomplete.js

```diff
@@ -76,6 +77,11 @@ export default function(state = INITIAL_STATE, action) {
         ...state,
         name: action.payload,
       };
+    case GEOCODE_COMPLETE:
```

Comment:
this action is almost the same as `UPDATE_AUTOCOMPLETE_LOCATION`

Reply:
Yeah except the GEOCODE_COMPLETE replaces `autocomplete.location` prop so the text can be editable after the user clicks the current location button.